src/git_changelog/commit.py

```diff
@@ -223,6 +254,11 @@ class BasicConvention(CommitConvention):
         TYPES["remove"],
     ]
 
+    @property
```

Comment:
Given that the `CommitConvention` classes utilize class attributes (rather than instance attributes), the process becomes slightly intricate. When updating self.TYPES, it's crucial for the regex to be updated as well. Consequently, we're presented with two options: either use a property or relocate it under `__init__`. I'm not sure which choice is superior.